README.md

```diff
@@ -120,4 +168,84 @@
 
 ## Improve local-gpu cluster usability [INFRA]
 
-- [ ] Improve local-gpu usability for the ML team so that it can be adopted as the main ML platform instead of science-3
\ No newline at end of file
+- [ ] Improve local-gpu usability for the ML team so that it can be adopted as the main ML platform instead of science-3
+
+## Define DevRel Strategy/Tools [DevRel]
+
+- [ ] Define DevRel strategy/tools
+- [ ] Hire one more team member (Tech Writer / Advocate)
+- [ ] Hire social media intern
+- [ ] Define DevRel budget
+- [ ] Execute sponsorships / attendance strategies for Q4 & 2019
+- [ ] Decide and document responsibilities for Website content updates
+- [ ] Find & Onboard Swag vendor
+- [ ] Define a schedule of integrated campaigns for Q4 and 2019Q1
+- [ ] Improve metrics dashboard to show actionable data (MoM, YoY, QoQ)
+
+### Define PR & BD Strategy [DevRel]
+
+- [ ] Pitch the "What's in your code?" campaign to the press and get publication in at least 2 publications
+- [ ] Get Eiso bylines published in at least 2 publications
+- [ ] Secure at least 3 analysts briefings or demos
+- [ ] Qualify at least 1 BD led opportunity for joint announcement i.e Microsoft,  GitHub / GitLab etc
+
+### Manage Read/Write Community [DevRel]
+
+- [ ] Decide on new blogging platform
+- [ ] Decide on new forum platform
+- [ ] Define metrics of success per social platform and decide which ones matter
+
+### Define & Setup Email Marketing strategy [DevRel]
+
+- [ ] Curate content and send newsletter bi-weekly
+- [ ] Newsletter reaches 300 (+100% growth) subscribers
+- [ ] Set up master drip campaign for leads
+- [ ] Write report on email performance with hindsights and recommendations
+
+### Define and Manage a Community / Sales Funnel [DevRel]
+
+- [ ] Refine Salesforce + Pardot MQL and SQL rules
+- [ ] GDPR compliance 
+- [ ] Map and report on conversion funnel with hindsights and recommendations
+
+## Advance MLonCode awareness [DevRel]
+
+### Bring 'Assisted Code Review' to developers in the open-source community [DevRel]
```

Comment:
Maybe this one should be under the top-level `# Bring 'Assisted Code Review' to developers in the open-source community`?

Reply:
Done